tests/test_jobqueue.py

```diff
@@ -348,7 +349,14 @@ def test_job_next_t_property(self, job_queue):
                 else:
                     assert job.next_t == t
 
-        sleep(0.03)
+        assert self.result == 1
+        sleep(0.02)
 
+        assert self.result == 2
         assert job1.next_t is None
         assert job2.next_t is None
+
```

Comment:
Can you put this in a separate `test_job_next_t_setter`? That way the output of the pytest is more precise.

Reply:
Moved to separated test.